### PR TITLE
french translations for the reset password bundle

### PR DESCRIPTION
**Pull request by jrushlow**
_Translations were AI Generated by AWS Translate Machine Learning Platform and need to be Validated by humans_

Includes all of the French translations for Symfony Cast's Reset Password Bundle in PR #1100

### CODE REVIEW

File: translations/validators+intl-icu.fr.xlf [hunk 1]
```diff
@@ -30,6 +30,24 @@
                 <source>comment.is_spam</source>
                 <target>Le contenu de votre commentaire est considéré comme du spam.</target>
             </trans-unit>
+
+            <trans-unit id="reset_password.request_type.email_field">
+                <source>reset_password.request_type.email_field</source>
+                <target>S'il vous plaît entrer votre email</target>
```
Comment (vsilvestre-accenture) [inline translations/validators+intl-icu.fr.xlf:36]: Veuillez saisir votre email

File: translations/messages+intl-icu.fr.xlf [hunk 2]
```diff
@@ -434,6 +434,63 @@
...
+                <target>Mot de passe oublié</target>
+            </trans-unit>
+            <trans-unit id="reset_password.title">
+                <source>reset_password.title</source>
+                <target>Réinitialiser votre mot de passe</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.check_email.p1">
+                <source>reset_password.check_email.p1</source>
+                <target>Un e-mail contenant un lien sur lequel vous pouvez cliquer pour réinitialiser votre mot de passe a été envoyé. Ce lien expirera dans {token} heure (s).</target>
```
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:449]: `heure (s).` =>  `heure(s).`

File: translations/messages+intl-icu.fr.xlf [hunk 3]
```diff
@@ -434,6 +434,63 @@
...
+                <source>reset_password.check_email.help_user_exists</source>
+                <target>Dans une application réelle, l'e-mail ci-dessous serait envoyé et l'utilisateur serait redirigé vers la route app_check_email.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_not_found">
+                <source>reset_password.check_email.help_user_not_found</source>
+                <target>Étant donné que l'adresse e-mail que vous avez saisie n'existe pas, nous affichons ce modèle, au lieu de révéler l'utilisateur n'existe pas.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.title">
+                <source>reset_password.check_email.title</source>
+                <target>Réinitialiser le mot de passe E-mail envoyé</target>
```
Comment (vsilvestre-accenture) [inline translations/messages+intl-icu.fr.xlf:465]: Email de réinitialisation de mot de passe envoyé

File: translations/messages+intl-icu.fr.xlf [hunk 4]
```diff
@@ -434,6 +434,63 @@
...
+                <source>reset_password.check_email.p2</source>
+                <target><![CDATA[Si vous ne recevez pas d'e-mail, veuillez vérifier votre dossier de spam ou <a href="{reset_password_link}">réessayer</a>.]]></target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_exists">
+                <source>reset_password.check_email.help_user_exists</source>
+                <target>Dans une application réelle, l'e-mail ci-dessous serait envoyé et l'utilisateur serait redirigé vers la route app_check_email.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_not_found">
+                <source>reset_password.check_email.help_user_not_found</source>
+                <target>Étant donné que l'adresse e-mail que vous avez saisie n'existe pas, nous affichons ce modèle, au lieu de révéler l'utilisateur n'existe pas.</target>
```
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:461]: `Étant donné que l'adresse e-mail que vous avez saisie n'existe pas, nous affichons ce template, au lieu de révéler que l'utilisateur n'existe pas.`

File: translations/messages+intl-icu.fr.xlf [hunk 5]
```diff
@@ -434,6 +434,63 @@
...
+                <target>Salut !</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.email.body">
+                <source>reset_password.email.body</source>
+                <target><![CDATA[Pour réinitialiser votre mot de passe, veuillez visiter <a href="{email_link}">ici</a> Ce lien expirera dans {expire} heure (s)..]]></target>
+            </trans-unit>
+            <trans-unit id="reset_password.email.closing">
+                <source>reset_password.email.closing</source>
+                <target>Les applaudissements !</target>
```
Comment (vsilvestre-accenture) [inline translations/messages+intl-icu.fr.xlf:478]: "Bien cordialement!" have more sense here to keep the "joy" expression

File: translations/messages+intl-icu.fr.xlf (hunk 3 above)
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:465]: `E-mail de réinitialisation de mot de passe envoyé`

File: translations/messages+intl-icu.fr.xlf [hunk 6]
```diff
@@ -434,6 +434,63 @@
...
+                <source>reset_password.check_email.help_user_not_found</source>
+                <target>Étant donné que l'adresse e-mail que vous avez saisie n'existe pas, nous affichons ce modèle, au lieu de révéler l'utilisateur n'existe pas.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.title">
+                <source>reset_password.check_email.title</source>
+                <target>Réinitialiser le mot de passe E-mail envoyé</target>
+            </trans-unit>
+            <trans-unit id="reset_password.email.title">
+                <source>reset_password.email.title</source>
+                <target>Salut !</target>
```
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:469]: `Bonjour !`
`Salut !` is too informal

File: translations/messages+intl-icu.fr.xlf (hunk 5 above)
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:478]: `Félicitation !`

File: translations/validators+intl-icu.fr.xlf (hunk 1 above)
Comment (MichaelDemeyer) [inline translations/validators+intl-icu.fr.xlf:36]: `Veuillez entrer votre e-mail`

File: translations/messages+intl-icu.fr.xlf [hunk 7]
```diff
@@ -434,6 +434,63 @@
...
+            </trans-unit>
+
+            <trans-unit id="reset_password.email.body">
+                <source>reset_password.email.body</source>
+                <target><![CDATA[Pour réinitialiser votre mot de passe, veuillez visiter <a href="{email_link}">ici</a> Ce lien expirera dans {expire} heure (s)..]]></target>
```
Comment (MichaelDemeyer) [inline translations/messages+intl-icu.fr.xlf:474]: `Pour réinitialiser votre mot de passe, veuillez cliquer <a href="{email_link}">ici</a> Ce lien expirera dans {expire} heure(s)..`

File: translations/messages+intl-icu.fr.xlf (hunk 2 above)
Comment (stof) [reply translations/messages+intl-icu.fr.xlf:449]: even better would be to the the ICU `plural` rules.

File: translations/validators+intl-icu.fr.xlf [hunk 8]
```diff
@@ -30,6 +30,24 @@
...
+                <source>reset_password.change_password_type.invalid_message</source>
+                <target>Les champs de mot de passe doivent correspondre.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.change_password_type.password_blank">
+                <source>reset_password.change_password_type.password_blank</source>
+                <target>Veuillez saisir un mot de passe</target>
+            </trans-unit>
+            <trans-unit id="reset_password.change_password_type.password_length">
+                <source>reset_password.change_password_type.password_length</source>
+                <target>Votre mot de passe doit comporter au moins {limit} caractères</target>
```
Comment (stof) [inline translations/validators+intl-icu.fr.xlf:49]: Missing period at the end of the sentence (in the previous one as well)

File: translations/messages+intl-icu.fr.xlf (hunk 6 above)
Comment (jrushlow) [reply translations/messages+intl-icu.fr.xlf:469]: I'm not sure about this one, I agree Hello sounds better, but maker generates "Hi!"

I will go through after words and update the maker templates then most likely reflect those change here.

File: translations/validators+intl-icu.fr.xlf (hunk 8 above)
Comment (jrushlow) [reply translations/validators+intl-icu.fr.xlf]: thanks

File: translations/messages+intl-icu.fr.xlf (hunk 3 above)
Comment (jrushlow) [reply translations/messages+intl-icu.fr.xlf]: I went with "E-mail de réinitialisation de mot de passe envoyé" as I believe that better reflects what is generated by maker.

Comment (jrushlow) [reply translations/messages+intl-icu.fr.xlf]: thanks

**Comment by AxelBriche** [comment]
Hello, why translation PR is not merged please?

**Comment by javiereguiluz** [comment]
Closing because the related #1100 proposal was closed. Sorry! And thank you!

**Pull request closed without merging**